spanner_orm/field.py

```diff
@@ -43,23 +43,43 @@ def grpc_type() -> spanner_v1.Type:
   def validate_type(value: Any) -> None:
     raise NotImplementedError
 
+  @staticmethod
+  def matches(type: str) -> bool:
+    raise NotImplementedError
+
+  @staticmethod
+  def support_length() -> bool:
+    raise NotImplementedError
+
```

Comment:
(nit) Please add docstrings for new public methods. https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
Done.